static/js/app.js

```diff
@@ -114,9 +115,15 @@ $(function () {
         updateUrl: function (url) {
             if (typeof url == 'undefined') {
                 if (app.infoWindow || app.discussionShown) return;
-                url = "/?" + this.getCurrentUrlParams();
+                //url = "/?" + this.getCurrentUrlParams();
+                url = "";
+            }else{
+                if (url.indexOf('?') != -1) {
+                    url = url.substring(0, url.indexOf('?'));
+                }
             }
-            Backbone.history.navigate(url, true);
+           // Backbone.history.navigate(url, true);
+            window.history.pushState('','','/')
```

Comment:
This ends up having the exact same URL many times in the history (try playing with https://anyway-unstable-pr-494.herokuapp.com/ and clicking 'back' several times to see). Not very helpful.